Helper/Checkout.php

```diff
@@ -80,6 +80,6 @@ public function validateTransactionAmount($transactionId)
             $this->setBreadTransactionAmount($info['adjustedTotal']);
         }
 
-        return (bool) ($breadAmount == $quoteTotal);
+        return (bool) ($breadAmount == $quoteTotal || (abs((int)$breadAmount - (int)$quoteTotal) <= 2));
```

Comment:
do you think we should have it split out into two like this, or just use the <= 2 comparison, since that also includes ==?